packages/x-components/package.json

```diff
@@ -22,7 +22,7 @@
     "test:unit-watch": "vue-cli-service test:unit --watch",
     "test:e2e": "vue-cli-service test:e2e",
     "test:e2e:ci": "vue-cli-service test:e2e --headless --browser chrome",
-    "test": "npm run lint && npm run test:unit",
+    "test": "npm run lint && npm run test:unit && npm run test:e2e:ci",
```

Comment:
I just realized we are running the lint 3 times for the x-components package.

1.  As part of the global `npm run lint` command
2. As part of the x-components `test` command
3. As part of the `x-components `npm run test:e2e:ci` command

Remove the `npm run lint` here, and set the `test:e2e:ci` command to `vue-cli-service --skip-plugins eslint test:e2e --headless --browser chrome`